docs/sources/shared/datasources/tempo-traces-to-profiles.md

```diff
@@ -0,0 +1,66 @@
+---
+headless: true
+labels:
+  products:
+    - enterprise
+    - oss
+---
+
+[//]: # 'This file documents the Traces to profile configure and usage for the Tempo data source.'
+[//]: # 'This shared file is included in these locations:'
+[//]: # '/grafana/docs/sources/datasources/tempo/configure-tempo-data-source.md'
+[//]: # '/website/docs/grafana-cloud/data-configuration/traces/traces-query-editor.md'
+[//]: #
+[//]: # 'If you make changes to this file, verify that the meaning and content are not changed in any place where the file is included.'
+[//]: # 'Any links should be fully qualified and not relative: /docs/grafana/ instead of ../grafana/.'
+
+<!-- # Trace to profiles  -->
+
+{{< docs/experimental product="Trace to profiles" featureFlag="traceToProfiles" >}}
+
+Using Trace to profiles, you can use Grafana’s ability to correlate different signals by adding the functionality to link between traces and profiles.
+
+**Trace to profiles** lets you link your Grafana Pyroscope data source to tracing data.
+When configured, this connection lets you run queries from a trace span into the profile data.
+
+There are two ways to configure the trace to profiles feature:
+
+- Use a simplified configuration with default query, or
+- Configure a custom query where you can use a template language to interpolate variables from the trace or span.
+
+To use trace to profile, navigate to **Explore** and query a trace. Each span now links to your queries. Clicking a link runs the query in a split panel. If tags are configured, Grafana dynamically inserts the span attribute values into the query. The query runs over the time range currently selected in **Explore**.
```

Comment:
The query runs over the time range of the (span start time - 60) to (span end time + 60 seconds).

Reply:
```suggestion
To use trace to profiles, navigate to **Explore** and query a trace. Each span now links to your queries. Clicking a link runs the query in a split panel. If tags are configured, Grafana dynamically inserts the span attribute values into the query. The query runs over the time range of the (span start time - 60) to (span end time + 60 seconds).
```